src/main/java/teammates/common/datatransfer/TeamEvalResult.java

```diff
@@ -94,18 +94,22 @@ public TeamEvalResult(int[][] submissionValues) {
     }
 
     /**
-     * Replaces all missing points (for various reasons such as 'not sure' or
-     * 'did not submit') with NA.
+     * Replaces all missing points (for various reasons such as 'not sure' with NSU and
+     * 'did not submit' with NA).
      */
     private int[][] sanitizeInput(int[][] input) {
         int teamSize = input.length;
         int[][] output = new int[teamSize][teamSize];
         for (int i = 0; i < teamSize; i++) {
             for (int j = 0; j < teamSize; j++) {
                 int points = input[i][j];
-                boolean pointsNotGiven = points == Const.POINTS_NOT_SUBMITTED
-                                         || points == Const.POINTS_NOT_SURE;
-                output[i][j] = pointsNotGiven ? NA : points;
+                if (points == Const.POINTS_NOT_SUBMITTED) {
+                    output[i][j] = NA;
+                } else if (points == Const.POINTS_NOT_SURE) {
```

Comment:
This condition is not needed; `NSU` and `Const.POINTS_NOT_SURE` are one and the same thing so the `else` clause suffices.
In fact from the previous code you can simply change `points == Const.POINTS_NOT_SUBMITTED || points == Const.POINTS_NOT_SURE` to `points == Const.POINTS_NOT_SUBMITTED` and it would have worked as expected.